src/ui/wxWidgets/I18N/pos/pwsafe_de.po

```diff
@@ -4435,7 +4435,7 @@ msgstr "Groß/Kleinschreibung beachtend]"
 
 #: ../../../ui/wxWidgets/SelectionCriteria.cpp:113
 msgid " were "
-msgstr " wobei "
+msgstr " vom "
```

Comment:
This change looks not fine. Please consider the following actual German text in the report:
Alle Einträge **wobei** verglichen mit entsprechenden Einträgen aus "..."
or
Einträge, deren Gruppe gleicht "..." [nicht Groß/Kleinschreibung beachtend] **wobei** verglichen mit entsprechenden Einträgen aus "..." 

If you now change "**wobei**" with "**vom**" the German translation is not really nice